commit 4201134f82d594ee2261cb2ad4c93309ad3601a8 (refs/changes/75/1775/1)
Author: Trent Mick <trentm@gmail.com>
Date:   2017-04-10T11:10:02-07:00 (2 years, 6 months ago)
    
    TOOLS-1750 node-sdc-clients test failure: imgapi.test.js failing "ping" test

diff --git a/test/imgapi.test.js b/test/imgapi.test.js
index 2db88ad..2992e4e 100644
--- a/test/imgapi.test.js
+++ b/test/imgapi.test.js
@@ -3,7 +3,10 @@ var test = require('tape');
 
 var IMGAPI = require('../lib/index').IMGAPI;
 
-var IMGAPI_URL = 'http://' + (process.env.IMGAPI_IP || '10.99.99.21');
+if (!process.env.IMGAPI_IP) {
+    throw new Error('IMGAPI_IP envvar is not set');
+}
+var IMGAPI_URL = 'http://' + process.env.IMGAPI_IP;
 
 var imgapi;
 
diff --git a/test/runtests b/test/runtests
index 63996da..c560e09 100755
--- a/test/runtests
+++ b/test/runtests
@@ -132,6 +132,7 @@ else
     export UFDS_ADMIN_UUID=$(bash /lib/sdc/config.sh -json | json ufds_admin_uuid)
     export DSAPI_URL=$(bash /lib/sdc/config.sh -json | json dsapi_url)
     export PAPI_IP=$(bash /lib/sdc/config.sh -json | json papi_admin_ips | cut -d, -f1)
+    export IMGAPI_IP=$(sdcadm insts imgapi -j | json 0.ip)
     export CNS_IP=$(sdcadm insts cns -j | json 0.ip)
     export MORAY_IP=$(sdcadm insts moray -j | json 0.ip)
 fi
